README.md

```diff
@@ -984,7 +985,60 @@ Each topic has two identifiers:
 
 ## 4.3 File Services
 
-### 4.3.1 GET Files (Header) Service
+### 4.3.1 GET Project Files Service
+
+**Resource URL**
+
+    GET /bcf/{version}/projects/{project_id}/files
+    
+[project_files_GET.json](Schemas_draft-03/Collaboration/File/project_files_GET.json)
+
+Retrieve a **collection** of project files that supports allowing users to choose which files (models) to reference in
+the header of topics ppostyed to that the server. 
```

Comment:
Typo in ppostyed

Reply:
Fixed, and documentation improved. Can you please check?